System preparation: hannah - stereochemistry and cofactors

hannahbruce opened the pull request.

hannahbruce commented:
#141

IAlibay reviewed (changes requested):
Request changes on the text for stereoisomers, and also possibly adding a suggested reference for that text.

fjclark commented:
Thanks for the contribution and review!

@hannahbruce, could you please address these comments?

fjclark commented:
Hi @hannahbruce, just a reminder to please make the minor changes requested above when you get a chance (although I appreciate that you're at the conference this week). Thank you!

fjclark commented:
Hi @hannahbruce, as most major changes have been made (and I'm going on leave today), we're trying to move everything to the Overleaf where we'll continue tweaking. I've merged all other PRs and synchronised this morning. If the conflicts are too much, would you mind making the changes again on the Overleaf, plus the changes suggested by @IAlibay? Apologies both, I realise this isn't optimal, but hopefully this works well enough given that only minor changes are needed.

Further comments could be added on the Overleaf, using https://github.com/alchemistry/alchemical-best-practices/issues/141 to ping each other.

Thanks very much.

hannahbruce commented:
Thanks @fjclark !

@IAlibay, I've moved the changes over to overleaf, if you're able to check there. I've added in both suggested references - thanks so much!

fjclark commented:
Great, thanks @hannahbruce.

Closing as it's been a couple of weeks and only minor changes were suggested (and were addressed).

Thanks both!
